docs/source/Evaluation_Tutorial.md

```diff
@@ -26,145 +26,79 @@ Once you know which dataset you want to train on, you can follow the instruction
+- `tb_log_dir`: GCS path to store tensorboard logs used to visualize the training progress.
+- `checkpoint_dir`: GCS path to store output estimator checkpoints. These checkpoints represent your trained model and will be used for evaluation.
 - `volume_size`: Size of the Kubernetes Persistent Volume Claims (PVC) that will be used to store the dataset. You can use the default value.
 
-> You'll want to change `tb_log_dir`, `checkpoint_dir` to point to a location that is convenient for you and your Kubernetes cluster have permissions to write to. This is typically a GCS path under the same GCP project. You want to keep a note on these directories that will be used for tensorboard visualization and model [evaluation](#part-3-evaluate-a-model). Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to these files inside a docker container.
+Set `tb_log_dir` and `checkpoint_dir` to a location that is convenient for you and your Kubernetes cluster has permissions to write to. This is typically a GCS path under the same GCP project. You want to keep a note on these directories that will be used for tensorboard visualization and model evaluation. Note that an invalid location will cause the job to fail, whereas a path to the local filesystem may run but will be hard to monitor as you won't have easy access to the files.
 
 ![pipeline graph](_images/kubeflow/train_pipeline_graph.jpg)
 
-To open tensorboard for training visualization, you can run the following command:
-
-```bash
-docker run \
-  -p 6006:6006 \
-  -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/key.json \
-  -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/key.json:ro \
-  -t tensorflow/tensorflow \
-  tensorboard \
-  --host=0.0.0.0 \
-  --logdir=gs://<tb_log_dir>
-```
-
-Open `http://localhost:6006` in web browser to see tensorboard results. This command assumes you have an environment variable `GOOGLE_APPLICATION_CREDENTIALS` in the host machine that points to a GCP service account credential. This service account should have permissions to read `tb_log_dir` to download tensorboard files. If you don't have a GCP service account credential, you should follow this [instruction](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application) to generate a valid credential.
+Next you can jump to [part 3](#part-3-monitor-training-in-tensorboard) to monitor the training in tensorboard. You'll need the locations of your logs and checkpoints in the next steps.
 
-Next, follow the [instructions](#part-3-evaluate-a-model) to evaluate the performance of this model by running one more pipeline we have prepared. You'll need the location of your model in the next step.
-
-### Train on UnityGroceries-RealWorld dataset (optional)
+### Train on UnityGroceries-RealWorld dataset
```

Comment:
I think we should move to optional section.

Reply:
Done